### PR TITLE
[RHOSINFRA-105] Swift scale up down

### PR DESCRIPTION
**Pull request by ovorobio**

### CODE REVIEW

File: playbooks/installer/ospd/post_install/scale_swift.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  gather_facts: no
+  roles:
+      - {role: "installer/ospd/overcloud/scale/", service: 'swift-storage', amount: 1}
+
+- name: Scaling swift-storage nodes up
+  hosts: undercloud
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  gather_facts: no
+  pre_tasks:
```
Comment (obaranov) [inline playbooks/installer/ospd/post_install/scale_swift.yml]: I'd suggest to add more descriptive comment why we need to wait for 60 minutes here.

File: playbooks/installer/ospd/post_install/scale_swift.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      - name: Wait for min_part_hours timeout
+        pause: minutes=60
+
+  roles:
+      - {role: "installer/ospd/overcloud/scale/", service: 'swift-storage', amount: 3}
```
Comment (obaranov) [inline playbooks/installer/ospd/post_install/scale_swift.yml]: Is it possible to take amount: 3 from topolgy (the number of swift nodes)? Or maybe make this setting overloadable? The same is for amount: 1

**Comment by yfried-redhat** [comment]
retest openstack

File: playbooks/installer/ospd/post_install/scale_swift.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  roles:
+      - {role: "installer/ospd/overcloud/scale/", service: 'swift-storage', amount: 1}
+
+- name: Scaling swift-storage nodes up
+  hosts: undercloud
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  gather_facts: no
+  pre_tasks:
+      - name: Wait for min_part_hours (the swift timeout between partition reassignment) timeout
```
Comment (yfried-redhat) [inline playbooks/installer/ospd/post_install/scale_swift.yml]: 60 min pause is problematic.
I would like to see a proper waiter that makes it clear what we are waiting for (partition creation? file creation? boot?) and not a hard coded pause. Please use `wait_for` module or `retry`

**Comment by yfried-redhat** [comment]
:-1: 
1. Please pull latest code to pass the gate
2. Please avoid hard coded determinate pause. Use a proper waiter instead.

**Pull request merged**